notebook2/notebook/notebook.py

```diff
@@ -338,15 +314,15 @@ def root():
 def notebook_page():
     notebooks = get_live_user_notebooks(user_id = user_id_transform(session['user']['id']))
 
-    if notebooks is None:
+    # https://github.com/hail-is/hail/issues/5487
+    assert len(notebooks) <= 1
```

Comment:
this will break if the user manages to submit 2, preventing them from ever using the service again.

Reply:
I'm working on fine-grained state change updates; once those are in we can drop this restriction.